app-cdr/burn-cd/burn-cd-1.8.1-r1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Please fix the copyright header, you can use repoman to commit so it updates this for you automatically.